src/components/SearchGridCell.vue

```diff
@@ -1,10 +1,14 @@
 <template>
-  <div>
+  <div class="search-grid_item-container"
+    :style="`width: ${containerAspect * 165}px;
```

Comment:
Could you move this `165` value to a constant, with an appropriate name please?

Reply:
Updated to a data value that's calculated with min row width / max aspect, so min img container width will not exceed min row width.

Reply:
Also the provider logo aspect ratio seems to be a problem with the original css? it has a 40 to 30 max dimension even though the logo is a square. Changing the dimension limit to 30 by 30 seems to solve the problem.